WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsDetailActivity.java

```diff
@@ -355,20 +354,23 @@ private void setActionBarTitleForNote(Note note) {
                     case NOTE_LIKE_TYPE:
                         title = getString(R.string.like);
                         break;
-                    case NOTE_COMMENT_TYPE:
-                        title = getString(R.string.comment);
-                        break;
                 }
             }
 
-            // Force change the Action Bar title for 'new_post' notifications.
+            // Force change the Action Bar title for 'new_post' and 'comment' notifications.
             if (note.isNewPostType()) {
                 title = getString(R.string.reader_title_post_detail);
+            } else if (note.isCommentType()) {
+                title = "";
             }
 
             getSupportActionBar().setTitle(title);
             // important for accessibility - talkback
-            setTitle(getString(R.string.notif_detail_screen_title, title));
+            if (title.isEmpty()) {
```

Comment:
Nit pick: Wdyt of using the `note.isCommentType()` condition on the if statement and remove the `else if` above?

Reply:
I just replaced the `if (title.isEmpty())` with `if (note.isCommentType())`, but we still need the `else if` for setting the variable `title` to an empty string. Because `getSupportActionBar().setTitle()` is different from  `setTitle()`